hobbymatcher/src/main/java/com/hobbymatcher/controller/event/EventsController.java

```diff
@@ -78,10 +78,33 @@ public EventsController(EventsService eventsService) {
     @ResponseBody
     public Map<String, Object> joinEvents(String id, @RequestParam(value = "events_id") String eventsId, HttpServletResponse response, HttpServletRequest request) {
         Map<String, Object> modelMap = new HashMap<String, Object>();
-        //User user = (User) request.getSession().getAttribute("user");
-        Boolean result = eventsService.joinEvents(id, eventsId);
-        modelMap.put("status", result);
-        response.setStatus(result ? 200 : 400);
+        try {
```

Comment:
An easier way to do this is would be to create a query to check if the specific record exists already. For example: `Select * from join_events where user_id = ${userid} and events_id = ${eventId}`

Reply:
Yeah I tried first that but there's something tricky in mybatis. The return result type is different than geteventsbyid. So I use the other way.